plugins/com.google.cloud.tools.eclipse.appengine.libraries/src/com/google/cloud/tools/eclipse/appengine/libraries/model/CloudLibraries.java

```diff
@@ -89,9 +89,11 @@ public static Library getLibrary(String id) {
       for (JsonObject api : apis) {
         String name = api.getString("name");
         String id = api.getString("id");
+        JsonArray transports = api.getJsonArray("transports");
         Library library = new Library(id);
         library.setGroup(CLIENT_APIS_GROUP);
         library.setName(name);
+        library.setTransport(transports.getString(0));
```

Comment:
Why throw away data?  It's not obvious that the `transports` array has a prioritized order.

Reply:
ATM there is exactly one transport per API. The JSON is a little more general because the API team said this may not always be true.

Reply:
That would be worth noting here in a comment.

Reply:
done